.changeset/dull-falcons-hope.md

```diff
@@ -0,0 +1,7 @@
+---
+'@swisspost/design-system-documentation': minor
+'@swisspost/design-system-components': minor
+'@swisspost/design-system-styles': minor
+---
+
+implement back-to-top button
```

Comment:
Please make sure to follow the guidelines for the changeset message: it should start with a title-cased verb in past tense and end with a period.